local.tf

```diff
@@ -0,0 +1,25 @@
+provider "random" {
+  version = "~> 2.1"
+}
+
+resource "random_string" "suffix" {
+  length  = 8
+  special = false
+}
+
+resource "random_pet" "current" {
+  prefix    = "tf-jx"
```

Comment:
What's the prefix good for? That's not part of the cluster name then, right?

Reply:
If we go with this, I'll need to do the same for GKE. There I use the hard coded 'jenkins-x'. Makes explaining things a bit easier, since you don't have to go into the random name generation business.  

If you like it, let's keep it. For me it feels more like a toy.

Reply:
This is exactly what we are doing for `jx create cluster`. If you don't provide a cluster name we'll generate one for you. I think this is an interesting feature that you can find almost anywhere where you don't want to provide configuration just to try it out.

The prefix makes it so the final name of the cluster will be: `tf-jx-<generated_name>`.